pos_product_multi_barcode/static/src/js/db.js

```diff
@@ -7,6 +7,27 @@ odoo.define("pos_product_multi_barcode.db", function (require) {
     models.load_fields("product.product", ["barcodes_json"]);
 
     PosDB.include({
+        _product_search_string: function (product) {
+            var str = product.display_name;
```

Comment:
I think it's possible to overload this function to call super instead of copy the all function and avoid other modules to overload correctly the function. Dont you think ?

Reply:
yes right.